music21/metadata/__init__.py

```diff
@@ -1644,13 +1603,13 @@ def title(self):
             'movementName',
         )
         for uniqueName in searchId:
-            titles = self._get(uniqueName, isCustom=False)
-            if not titles:  # get first matched
-                continue
-
-            return self._getStringValueByNamespaceName(
+            titleSummary: t.Optional[str] = self._getStringValueByNamespaceName(
                 properties.UNIQUE_NAME_TO_NAMESPACE_NAME[uniqueName]
             )
+            if not titleSummary:  # get first matched
+                continue
+
+            return titleSummary
```

Comment:
small point.  Why not:  

```
if titleSummary:
    return titleSummary  # return first matched
```

with no need for a continue, since it's the end of the loop.

Reply:
I like that better.

Reply:
Done.